content/general/concepts/github/github.md

```diff
@@ -0,0 +1,17 @@
+---
+Title: 'Github'
+Description: 'Github is a UI based version control that uses git.'
```

Comment:
```suggestion
Description: 'Github is a UI-based version control that uses Git.'
```

Unless we're referring to a specific command, let's change all instances of "git" to "Git".